### DISCUSSION TITLE
How to use this tools if there are more than one requirements.txt in the project

### DISCUSSION THREAD

**Discussion by wqh17101** [CycloneDX/cyclonedx-python#483] [Q&A] [answered]
How to use this tools if there are more than one requirements.txt in the project

 cyclonedx-py --r -i "path1" can only specify one requirements.txt

**Comment by jkowalleck** [comment]
it totally depends on the purpose of your multiple `requirements.txt` files. 
There is no general answer. 

One thing you could always do: create multiple SBOM results and merge them with https://github.com/CycloneDX/cyclonedx-cli

**Comment by jkowalleck** [comment, marked as answer]
FYI: the upcoming v4 will have various ways to support multiple `requirements.txt`.
this includes examples like
-  `cat req1.txt req2.txt | cyclonedx-py requirements`
-  `cyclonedx-py requirements requirements-dev.txt` where the requirements-file internally uses `-r deep concats`
